### PR TITLE
fix first and last for negative row count

### PR DESCRIPTION
**Pull request by bkamins**
Reported on Slack

### CODE REVIEW

File: src/abstractdataframe/abstractdataframe.jl [hunk 1]
```diff
@@ -558,14 +558,19 @@ Base.first(df::AbstractDataFrame) = df[1, :]
     first(df::AbstractDataFrame, n::Integer; view::Bool=false)
 
 Get a data frame with the `n` first rows of `df`.
+If `n` is greater than number of rows of `df` get all rows.
```
Comment (nalimilan) [inline src/abstractdataframe/abstractdataframe.jl:561]: For consistency with other sentences:
```suggestion
Get all rows if `n` is greater than the number of rows in `df`.
```

File: src/abstractdataframe/abstractdataframe.jl [hunk 2]
```diff
@@ -580,14 +585,19 @@ Base.last(df::AbstractDataFrame) = df[nrow(df), :]
     last(df::AbstractDataFrame, n::Integer; view::Bool=false)
 
 Get a data frame with the `n` last rows of `df`.
+If `n` is greater than number of rows of `df` get all rows.
```
Comment (nalimilan) [inline src/abstractdataframe/abstractdataframe.jl:588]:
```suggestion
Get all rows if `n` is greater than the number of rows in `df`.
```

**Review by nalimilan — APPROVED** [review]
This is technically breaking, but I guess we can consider this as a bug fix.

**Comment by bkamins** [comment]
> This is technically breaking

I considered it a bug, because the contract in Base disallows negative values and we should follow this contract.

**Comment by bkamins** [comment]
Thank you!

**Pull request merged**